astropy/io/fits/fitstime.py

```diff
@@ -0,0 +1,233 @@
+                                                             global_info['loc_z'], 
+                                                             unit='m')
+        except KeyError:
+            pass
+
+def FITS_to_Time(hdr, table):
+    """
+    Read FITS binary table time columns as `~astropy.time.Time`.
+
+    This method reads the metadata associated with time coordinates, as 
+    stored in a FITS binary table header, converts time columns into 
+    `~astropy.time.Time` columns and reads global reference times as 
+    `~astropy.time.Time` instances.
+
+    Parameters
+    ----------
+    hdr : `~astropy.io.fits.header.Header`
+        FITS Header
+    table : astropy.table.Table
+        The table whose time columns are to be read as Time
+
+    Returns
+    -------
+    hdr : `~astropy.io.fits.header.Header`
+        Modified FITS Header (time metadata removed)
+    """
+
+    # Set defaults for global time scale, reference, etc.
+    global_info = {'scale' : 'UTC',
+                   'mjdref' : None,
+                   'unit' : 's'}
+
+    # Set default dictionary for time columns
+    time_columns = defaultdict(OrderedDict)
+
+    # Make a "copy" (not just a view) of the input header, since it
+    # may get modified.  the data is still a "view" (for now)
+    hcopy = hdr.copy(strip=True)
+
+    for key, value, comment in hdr.cards:
+        if (key in TIME_KEYWORDS):
+
+            global_info[TIME_KEYWORDS[key]] = value
+            hcopy.remove(key)
+
+        elif (is_time_column_keyword(key)):
+
+            base, idx = re.match(r'([A-Z]+)([0-9]+)', key).groups()
+            time_columns[int(idx)][COLUMN_TIME_KEYWORDS[base]] = value
+            hcopy.remove(key)
+
+    if len(hcopy) != len(hdr):
+        convert_time_columns(table, global_info, time_columns)
+
+    return hcopy
+
+
+def Time_to_FITS(table):
+    """
+    Replace Time columns in a Table with non-mixin columns containing
+    each element as a vector of two doubles (jd1, jd2) and return a FITS 
+    header with appropriate time coordinate keywords.
+    jd = jd1 + jd2 represents time in the Julian Date format with 
+    high-precision.
+
+    Parameters
+    ----------
+    table : astropy.table.Table
+        The table whose Time columns are to be replaced.
+
+    Returns
+    -------
+    table : astropy.table.Table
+        The table with replaced Time columns
+    hdr : `~astropy.io.fits.header.Header`
+        Header containing Cards associated with the FITS time coordinate
+    """
+
+    # Shallow copy of the input table
+    newtable = table.copy(copy_data=False)
+
+    # Global time coordinate frame keywords
+    hdr = Header([Card(keyword=key, value=val[0], comment=val[1]) 
+                  for key, val in GLOBAL_TIME_INFO.items()])
+
+    time_cols = table.columns.isinstance(Time)
+
+    # Geocentric Position
+    pos_geo = None
+
+    for col in time_cols:
+        jd12 = np.array([col.jd1, col.jd2])
+        jd12 = np.rollaxis(jd12, 0, jd12.ndim)
```

Comment:
This is a bit obscure and deserves comment.  I'm guessing this is about the default Fortran byte ordering in FITS?